app/src/main/java/org/simple/clinic/security/pin/PinEntryCardView.kt

```diff
@@ -65,7 +65,7 @@ class PinEntryCardView(context: Context, attrs: AttributeSet) : CardView(context
         events = events.ofType(),
         defaultModel = PinEntryModel.default(),
         init = PinEntryInit(),
-        update = PinEntryUpdate(),
+        update = PinEntryUpdate(submitPinAtLength = 4),
```

Comment:
Technically, this is purely an UI concern since the number of digits in the PIN is decided in the UI, and the only thing we do with that value is decide when to submit the PIN. Ideally, I'd like this PIN length to be hardcoded in the UI and it just auto submits the PIN entered event when the user enters 4 digits.

Reply:
I was thinking I'll make this change later once I complete the entire user API migration to v4.